src/api/attach/attach-hooks.ts

```diff
@@ -11,17 +11,23 @@ type UploadFiles = {
     formData: FormData;
 };
 
-export const useAttachesCreateMutation = (): UseMutationResult<Attach, unknown, UploadFiles> => {
+export const onUploadSuccess = (sectionId: number) => {
```

Comment:
Technically it is a hook, but it's name does not start with "use". I think onSuccess and onFail should be useCallback functions on the section feedback page